### PR TITLE
[E2E] Add test for Java Vertx workspace.

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Add test for Java Vertx default worskpace with example project. 
Test will validate:
 - Workspace creation via UI
 - Language server validation
   - Expand project and open file in editor
   - Java LS initialization
   - Suggestion invoking
   - Error highlighting
   - Autocomplete
   - Codenavigation
 - Validation of project build
 - Close the terminal tasks
 - Stop workspace
 - Delete workspace

Test can be run via `npm run test-java-vertx`.

EDIT: The purpose of this test is to test default example stacks with their example project that user can create just by selecting it in UI.

### CODE REVIEW

**Comment by Katka92** [comment]
ci-test

**Comment by Katka92** [comment]
ci-build

File: e2e/tests/e2e/JavaVertx.spec.ts [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        });
+
+        test('Wait project imported', async () => {
+            try {
+                await projectTree.waitProjectImported(sampleName, 'src');
+            } catch (err) {
+                if (!(err instanceof error.TimeoutError)) {
+                    throw err;
+                }
+                var rest: restClient.RestClient = new restClient.RestClient('user-agent');
```
Comment (musienko-maxim) [inline e2e/tests/e2e/JavaVertx.spec.ts]: Why do we use var?

Comment (Katka92) [reply e2e/tests/e2e/JavaVertx.spec.ts]: Seems like we don't need this try/catch block at all. I'll remove it.

Comment (Katka92) [reply e2e/tests/e2e/JavaVertx.spec.ts]: Changes pushed. Thank you for valuable comment!

**Review by musienko-maxim — APPROVED** [review]

**Comment by Katka92** [comment]
ci-test

**Comment by dmytro-ndp** [comment]
@Katka92: If there is an issue which this PR fixes?

**Comment by dmytro-ndp** [comment]
@Katka92: do we need  special devfile to create test workspace?

**Comment by Katka92** [comment]
@dmytro-ndp There is an issue in Rh-Che [1] and we think that this logic should be placed in upstream as there is nothing Rh-Che specific in a test.
The purpose of this test is to test default example stacks with their example project, so there is no special devfile used. We plan to extend our test suite to be able to test majority of workspaces that user can create just by selecting them in UI.

[1] https://github.com/redhat-developer/rh-che/issues/1567

**Comment by dmytro-ndp** [comment]
@Katka92: thanks for the answer! IMHO, the answer could be added to description of PR to explain what it is for.
I have just also realized that file `mocha-java-vertx.opts` which is required to run the test suite is absent. Is it expected?

**Comment by Katka92** [comment]
@dmytro-ndp Good catch! I forgot to add this new file, thank you very much! I've also updated PR description.

**Review by dmytro-ndp — APPROVED** [review]
Looks good.

**Pull request merged**